CHANGELOG.md

```diff
@@ -4,123 +4,212 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
```

Comment:
@zanieb if you're going to use that blurb from the other comment in the release, then this is the only thing left to change. If you don't get to this today, you should be able to make edits to this PR so set this date correctly. 

```suggestion
## 0.8.0 (August 23rd, 2023)
```

Reply:
```suggestion
## 0.8.0 (August 28th, 2023)
```